Navigation.css

```diff
@@ -0,0 +1,98 @@
+:root{
+    --primary-color:turquoise;
+    --secondary-color:rgb(155, 173, 238);
+    --tertiary-color: skyblue;
+
+}
+
+body{
+    position: relative;
+    margin-left: 4px;
+    width: 100%;
+}
+
+.body{
+    width: 100%;
+}
+.container{
+    padding: 3px;
+    display: flex;
+    gap: 5px;
+    position: relative;
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    margin: 5px;
+    background-color: var(--secondary-color);
+    border-bottom-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
```

Comment:
suggestion: you can use shorthand property too
![image](https://user-images.githubusercontent.com/69811810/153249402-b8c7d6a7-fe39-4de4-bc18-a4329ff553ce.png)